osmdroid-android/src/main/java/org/osmdroid/views/overlay/mylocation/MyLocationNewOverlay.java

```diff
@@ -295,7 +296,10 @@ public boolean getEnableAutoStop(){
 	@Override
 	public boolean onTouchEvent(final MotionEvent event, final MapView mapView) {
 		if (event.getAction() == MotionEvent.ACTION_DOWN && enableAutoStop) {
-			this.disableFollowLocation();
+      CustomZoomButtonsController zoomController = mapView.getZoomController();
+      if ( zoomController != null && !zoomController.isHandling(event) ) {
+        this.disableFollowLocation();
+      }
```

Comment:
Too intrusive!

I think the solution would be something like adding those lines in `MapView.dispatchTouchEvent`, just after the "old zoom controller touch test":
````java
if (mZoomController != null && mZoomController.isTouched(event)) {
	return true;
}
````
And additionally make `CustomZoomButtonsController.isTouched` `public` and making it an always non-rotated version.

Could you have a go with that? My first test looks OK, but as the "follow" feature doesn't work on my old smartphone I cannot test it.

Reply:
Thx @monsieurtanuki. 

I was looking for an less intrusive place, but didn't find it. I will check your suggestion! And come back later.

The `CustomZoomButtonsController.isTouched()` thing did not work! As `isTouched()` right now also immediately executes the touch, this is why I introduced that `isHandling()` thing doing the mere check without side effect. Might of course be unnecessary, when I can get the events really early. 

I will look into both.

Reply:
Hmm, am I supposed to close this change discussion?